### PR TITLE
Fix nil organizations in home page feed endpoint

### PR DESCRIPTION
**Pull request by joshpuetz**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [x] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description
Fixes a issue with home feed json generation that occurred when an article doesn't have an attached organization. Thanks for @nickytonline for finding it!

## Added tests?

- [x] yes
- [ ] no, because they aren't needed
- [ ] no, because I need help

## Added to documentation?

- [ ] docs.dev.to
- [ ] readme
- [x] no documentation needed

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # we use this to be able to increase the size of the seeded DB at will
 # eg.: `SEEDS_MULTIPLIER=2 rails db:seed` would double the amount of data
-SEEDS_MULTIPLIER = [1, ENV["SEEDS_MULTIPLIER"].to_i].max
+SEEDS_MULTIPLIER = [6, ENV["SEEDS_MULTIPLIER"].to_i].max
```
Comment (nickytonline) [inline db/seeds.rb]: This was my bad. I checked this in by accident. It should be 1. I'll update it.

File: spec/requests/stories/feeds_spec.rb [hunk 2]
```diff
@@ -4,27 +4,55 @@
   let(:title) { "My post" }
   let(:user) { create(:user, name: "Josh") }
   let(:organization) { create(:organization, name: "JoshCo") }
-  let!(:article) { create(:article, title: title, featured: true, user: user, organization: organization) }
+  let(:tags) { "alpha, beta, delta, gamma" }
+  let(:article) { create(:article, title: title, featured: true, user: user, organization: organization, tags: tags) }
+
+  before do
```
Comment (citizen428) [inline spec/requests/stories/feeds_spec.rb]: Would it be easier to use `let!(:article)`, which forces method invocation before each example? And if we don't need to re-create the article before each spec, you can use the [`let_it_be`](https://github.com/palkan/test-prof/blob/master/docs/let_it_be.md) helper like we do in other places. 😃

Comment (joshpuetz) [reply spec/requests/stories/feeds_spec.rb]: I had no idea about `test_prof` and `let_it_be`, thanks for the heads up!

I actually do want to recreate the article before each spec (or at least before each context): there are different options we want to have on the article for each context. The issue with using `let!` is sequencing with the setting of other attributes we're setting on article.

Comment (citizen428) [reply spec/requests/stories/feeds_spec.rb]: Thanks for clarifying @joshpuetz 😃

**Pull request merged**